### DISCUSSION TITLE
Authenticator and mozilla.org / Firefox Account

### DISCUSSION THREAD

**Discussion by harl** [Authenticator-Extension/Authenticator#614] [General]
Hey.
As much as I've tried to get Authenticator to scan the QR code shown by the mozilla/Firefox account page when setting up the 2-factor authentication, I can't get it to to work even though it's able to create the white overlay on said page. The overlay just disappears after "enframing" the QR code without any kind of feedback. Some message or notification would be nice.
I take it this doesn't work for about the same reasons other add-ons don't work on mozilla's pages (and certain local addresses)?

**Comment by Sneezry** [comment]
I guess this is a known issue and there's a workaround: https://github.com/Authenticator-Extension/Authenticator/issues/610#issuecomment-797193104
